web-client/app/scripts/filters/timeToRead.js

```diff
@@ -3,6 +3,16 @@
 angular.module('webClientApp')
   .filter('timeToRead', [function () {
     return function(text) {
-      return Math.max(String(text).split(' ').length/200, 1);
+      var time =  Math.max(String(text).split(' ').length/200, 1).toFixed(0);
+      switch(true) {
```

Comment:
I've never seen this usage of the switch statement and it looks somewhat alien. I'd instead rewrote this using if-else statements. Also I would cast the time to integer number after calling toFixed (which will convert it to string).